onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -346,8 +464,13 @@ Return Value:
                         RowsRemaining, CountN, ldc, RowSums, ColumnSumBuffer,
                         DepthValue, ZeroMode);
 
-                    if (PostProcess && WorkBlock->CIsFloat) {
-                        MlasGemmU8X8OutputFloat(WorkBlock, c, n, RowsHandled, CountN);
+                    if (PostProcess && WorkBlock->OutputProcessor) {
```

Comment:
nit: You fixed the case above about OutputProcessor == nullptr, but didn't fix the other cases in the function. My comment was directed at all instances of that pattern.